app/views/claims/_school_search.html.erb

```diff
@@ -12,8 +12,11 @@
 
     <%= form_group_tag current_claim, :school_search do %>
 
+  </h1>
       <h1 class="govuk-label-wrapper">
+        <label class="govuk-label govuk-label--xl" for="school_search">
```

Comment:
I think this needs removing, when I look at the page there are two labels pointing at the same id.

Reply:
> I think this needs removing, when I look at the page there are two labels pointing at the same id.

Where is the second `<label>` element? 

The missing label is what I thought I was fixing and there was another story with the selection of the autocomplete issue we are waiting on the upstream fix?

https://trello.com/c/MEWC6cq0

Sorry if I am being thick!

Reply:
Seen the second label! 👍  other question still stands! :)

Reply:
Hmmm, going back and looking at the story I am wondering if we close this and cover all the autocomplete issues together? All this does now is add the focus state styles?

Reply:
Yep, let's close this and pick the autocomplete issue up later.